be/src/connector/file_connector.cpp

```diff
@@ -107,7 +107,7 @@ Status FileDataSource::get_next(RuntimeState* state, vectorized::ChunkPtr* chunk
         size_t before_rows = (*chunk)->num_rows();
 
         const TQueryOptions& query_options = state->query_options();
-        if (query_options.__isset.load_job_type && query_options.load_job_type == TLoadJobType::Broker) {
+        if (query_options.__isset.load_job_type && query_options.load_job_type == TLoadJobType::BROKER) {
```

Comment:
It's better to add a concise message about the feature in the commit message and related issues.